src/room.ts

```diff
@@ -66,4 +65,9 @@ export class Room {
 			conn.sendMessage(msg);
 		}
 	}
+
+	public broadcast(text: string) {
+		const msg: SystemMessage = {type: 'SYSTEM', message: text};
```

Comment:
Add a `level` or `severity` property to system messages. We might want to color-code some messages differently, although I can't come up with a tangible example right now. All system messages (the ones implemented here) should have `level: 'info'`.